modules/logging-loki-reliability-hardening.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-loki.adoc
+
+:_content-type: REFERENCE
```

Comment:
This module seems a little more `CONCEPT` than `REFERENCE` possibly? Although the "Configuring..." title and the last bit is more procedure-y sounding, too, so I'm not sure.

Reply:
ACK, I'd say it's actually more conceptual. I was thinking reference just because we give that config example in there, but really it's just explaining the concept of podaffinity for loki and that you can configure it, rather than giving any real step by step / sample code for configuring it. It would probably be another procedure module if we wanted to explain how users configure specific rules, but that's outside the scope.
TL;DR, I'll update this to a `CONCEPT` 🙂